perf/conf/tachyon-perf-env.sh.template

```diff
@@ -31,6 +31,9 @@ TACHYON_PERF_SHARED_FS="false"
 
 PERF_CONF_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
+# Specify this value and add to OPTS to test against another fs client impl.
+# TACHYON_PERF_UFS="Tachyon"
```

Comment:
May need an example, like "add '-Dtachyon.perf.ufs=$TACHYON_PERF_UFS' to TACHYON_PERF_JAVA_OPTS".